### PR TITLE
add new modules to fedora spec file

### PR DESCRIPTION
**Pull request by anandaravuri**

### CODE REVIEW

File: tools/libboard/board_a10gx/CMakeLists.txt [hunk 1]
```diff
@@ -31,3 +31,7 @@ opae_add_module_library(TARGET board_a10gx
         opae-c
     COMPONENT opaeboardlib
 )
+target_include_directories(board_a10gx
+    PRIVATE
+        ${OPAE_SDK_SOURCE}/ibboard/board_common
```
Comment (tswhison) [inline tools/libboard/board_a10gx/CMakeLists.txt:36]: s,ibboard,libboard,
Is this needed?  Seems to be building without it.

File: tools/libboard/board_d5005/CMakeLists.txt [hunk 2]
```diff
@@ -32,3 +32,8 @@ opae_add_module_library(TARGET board_d5005
         board_common
     COMPONENT opaeboardlib
 )
+
+target_include_directories(board_d5005
+    PRIVATE
+        ${OPAE_SDK_SOURCE}/ibboard/board_common
```
Comment (tswhison) [inline tools/libboard/board_d5005/CMakeLists.txt:38]: s,ibboard,libboard,
Is this needed?  Seems to be building without it.

File: tools/libboard/board_n3000/CMakeLists.txt [hunk 3]
```diff
@@ -32,3 +32,8 @@ opae_add_module_library(TARGET board_n3000
         board_common
     COMPONENT opaeboardlib
 )
+
+target_include_directories(board_n3000
+    PRIVATE
+        ${OPAE_SDK_SOURCE}/ibboard/board_common
```
Comment (tswhison) [inline tools/libboard/board_n3000/CMakeLists.txt:38]: s,ibboard,libboard,
Is this needed?  Seems to be building without it.

File: tools/libboard/board_d5005/CMakeLists.txt (hunk 2 above)
Comment (anandaravuri) [reply tools/libboard/board_d5005/CMakeLists.txt]: fixed libboard path

File: tools/libboard/board_n3000/CMakeLists.txt (hunk 3 above)
Comment (anandaravuri) [reply tools/libboard/board_n3000/CMakeLists.txt]: fixed libboard path

File: tools/libboard/board_a10gx/CMakeLists.txt (hunk 1 above)
Comment (anandaravuri) [reply tools/libboard/board_a10gx/CMakeLists.txt]: fixed libboard path

**Pull request merged**